src/packages/sdk/src/__tests__/engine-commands/getConfig.test.ts

```diff
@@ -0,0 +1,101 @@
+import stripAnsi from 'strip-ansi'
+import { getConfig } from '../..'
+
+describe('getConfig', () => {
+  test('empty config', async () => {
+    const config = await getConfig({
+      datamodel: `
+      datasource db {
+        provider = "sqlite"
+        url      = "file:../hello.db"
+      }
+      
+      model A {
+        id Int @id
+        name String
+      }`,
+    })
+
+    expect(config).toMatchSnapshot()
+  })
+
+  test('sqlite and createMany', async () => {
+    expect.assertions(1)
+    try {
+      await getConfig({
+        datamodel: `
+      datasource db {
+        provider = "sqlite"
+        url      = "file:../hello.db"
+      }
+
+      generator client {
+        provider = "prisma-client-js"
+        previewFeatures = ["createMany"]
+      }
+      
+      model A {
+        id Int @id
+        name String
+      }`,
+      })
+    } catch (e) {
+      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
+        "Get config: Database provider \\"sqlite\\" and the preview feature \\"createMany\\" can't be used at the same time.
+          Please either remove the \\"createMany\\" feature flag or use any other database type that Prisma supports: postgres, mysql or sqlserver."
+      `)
+    }
+  })
```

Comment:
Related for info:

createMany is GA since 2.20.0 and doesn't need a preview flag anymore.

We probably can remove this error?
https://github.com/prisma/prisma/blob/5a4d80ced4c8a9229d39bf38fec0ee87075a73ca/src/packages/sdk/src/engine-commands/getConfig.ts#L54-L62